app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,55 @@
+ <h1>Welcome to the Dashboard <%= @user.name %> </h1>
```

Comment:
This would also be a great place to go back and refactor with a partial.  I'd really like to do that if we have time.

Reply:
Let's add it to the refactor list -- I agree, partials would be great to incorporate